recipes/gnuastro/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set name = "gnuastro" %}
+{% set version = "0.10" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: http://ftp.gnu.org/gnu/{{name}}/{{name}}-{{version}}.tar.gz
+  sha256: 6d56dc161ecd80c64dbf09808f58114b52ad2ce648c71551b871f5d10d4c07c5
+
+build:
+  number: 0
+  skip:  True  # [win or osx]
```

Looks like the macOS error is due to `AT_FDCWD` being added in OSX 10.10 (by default conda-forge targets 10.9). It might be easy to patch in the source? If not can add a file like [this](https://github.com/conda-forge/qt-feedstock/blob/master/recipe/conda_build_config.yaml) to target 10.10.
```suggestion
  skip: True  # [win]
```

---

I would change the target to 10.10 as a last resort and just patch the source.

---

I think neither of us have access and/or experience with OSX, that's why we left it out.

@mohammad-akhlaghi would you be able to patch the source accordingly?